images/dns-controller/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:jessie
+FROM gcr.io/google_containers/ubuntu-slim:0.6
 
 # ca-certificates: Needed to talk to EC2 API
 RUN apt-get update && apt-get install --yes ca-certificates
```

Comment:
Should we install and use `localepurge`?  Also I usually do a house cleaning like:

```
        && apt-get clean \
        && rm -rf \
              doc \
              man \
              info \
              locale \
              /var/lib/apt/lists/* \
              /var/log/* \
              /var/cache/debconf/* \
              common-licenses \
              ~/.bashrc \
              /etc/systemd \
              /lib/lsb \
              /lib/udev \
              /usr/share/doc/ \
              /usr/share/doc-base/ \
              /usr/share/man/ \
              /tmp/*
```